requirements/use-cases.md

```diff
@@ -15,18 +15,29 @@ Regarding ordinary waste collection, the main actors are citizens, trucks and du
 
```

Comment:
Metto il commento qui ma in realtà mi riferisco alla riga sopra che non posso commentare perchè non è stata modificata. Abbiamo riusato la stessa descrizione dell' Ordinary Waste Collection ma in questo caso i Truck non sono coinvolti quindi proporrei di modificarla così:
```suggestion
Regarding ordinary waste disposal, the main actors are citizens and dumpsters. 
```